content/ew-issue-z200-[2017-03-29].yaml

```diff
@@ -101,30 +125,68 @@ content:
                 description: ""
                 link: https://www.youtube.com/watch?v=bhJ6YzBIoWo
+            -
+                title: "Audacious Enterprise GIS Apps with Ember.js"
+                description: ""
+                link: https://www.youtube.com/watch?v=7Bvr7cEQNCg
+    
+    -
+        heading: "So Ember 2017 Videos"
+        headlines:
+            -
+               title: "Keynote - Luke Melia"
+               description: ""
+               link: https://www.youtube.com/watch?v=FJp2f0VHSzU
+            -
+               title: "A journey through Ember.js glue - Mike North"
+               description: ""
+               link: https://www.youtube.com/watch?v=Femu3l8eS7U
+            -
+               title: "DOMless* Rendering with Ember - Miguel Andrade"
+               description: ""
+               link: https://www.youtube.com/watch?v=nkiZ_cWoQMk
+            -
+               title: "Failing Fast: Haskell Inspired Error Handling in Ember - Daniel Chappel"
+               description: ""
+               link: https://www.youtube.com/watch?v=BssIi9TDYFI
+            -
+               title: "Lazy Loading Engines Anything but Lazy - Trent Willis"
+               description: ""
+               link: https://www.youtube.com/watch?v=7ZP9-AIvrBU
+            -
+               title: "Building Accessible Applications: A Reasonable Approach - Melanie Sumner"
+               description: ""
+               link: https://www.youtube.com/watch?v=qxvNXBT03AY
 
 
     -
         heading: "Events"
         headlines:
             - 
-                title: Columbus - 4th April
-                link: "https://www.meetup.com/ember-columbus/events/238743363/"
-            - 
-                title: Seattle - 7th April
+                title: Seattle - 6th April
                 link: "https://www.meetup.com/Ember-js-Seattle-Meetup/events/238136145/"
             - 
-                title: Ember.JS DC - 7th April
+                title: Washington DC - 6th April
                 link: "https://www.meetup.com/Ember-JS-DC/events/237739025/"
             - 
-                title: Toronto - 7th April
+                title: Toronto - 6th April
                 link: "https://www.meetup.com/Toronto-Ember-JS-Meetup/events/238755300/"
             - 
-                title: Durham - 11th April
+                title: Raleigh - 10th April
                 link: "https://www.meetup.com/Triangle-Ember/events/235966115/"
             - 
-                title: Atlanta - 12th April
+                title: Atlanta - 11th April
                 link: "https://www.meetup.com/Ember-Atlanta-Meetup/events/238731803/"
-
+            -
+                title: Chicago - 12th April
+                link: "https://www.meetup.com/Chicago-Ember-js/events/237551101/"
+            - 
+                title: San Diego - 12th April
+                link: "https://www.meetup.com/sandiego-ember/events/wkkswlywgbqb/"
```

Comment:
So I realized that these dates were slightly off. This is because of my bad scrappy code in the meetup generator script. See https://github.com/ember-weekly/ew-meetup-generator/issues/2. I've never looked back at the code properly, but it should be a way of getting or calculating the local date of the meetup.